lib/kubernetes-deploy/render_task.rb

```diff
@@ -69,10 +69,10 @@ def render_filename(filename, stream)
       @logger.info("Rendering #{file_basename}...")
       file_content = File.read(File.join(@template_dir, filename))
       rendered_content = @renderer.render_template(filename, file_content)
-      implicit = true
-      YAML.parse_stream(rendered_content, "<rendered> #{filename}") { |d| implicit = d.implicit }
+      implicit = []
+      YAML.parse_stream(rendered_content, "<rendered> #{filename}") { |d| implicit << d.implicit }
```

Comment:
Do we need to change anything related to this here as well or that doesn't matter?

https://github.com/Shopify/kubernetes-deploy/blob/3d5cf5964932658a91e3e5621719ffaaa7ee0aac/lib/kubernetes-deploy/renderer.rb#L59-L62

Reply:
This bug happens because we take all the resources we're provided and turn it into a single output stream. So its important to have separators between each resource. The code you linked it for erb rendering a single file (which can contain partials) but its important that we render it as requested by the user.